coala_quickstart/generation/Bears.py

```diff
@@ -200,38 +200,48 @@ def get_non_optional_settings_bears(bears):
 
 def remove_unusable_bears(bears, unusable_bears):
     """
-    From the bears dict, filter the bears appearing in unusable_bears.
+    From the bears dict, filter the bears appearing in unusable_bears
+    and save them under “unusable” key for printing later.
 
     :param bears:
         A dict with language name as key and bear classes as value.
     :param unusable_bears:
         A collection of Bear classes.
     """
-    for language, language_bears in bears.items():
+    for language, language_bears in bears['usable'].items():
         for bear in tuple(language_bears):
             if bear in unusable_bears:
-                bears[language].remove(bear)
+                bears['usable'][language].remove(bear)
+                bears['unusable'][language] = bears['unusable'].get(
+                    language, ()) + (bear, )
 
 
-def print_relevant_bears(printer, relevant_bears, label='relevant'):
+def print_relevant_bears(printer, relevant_bears):
     """
-    Prints the relevant bears in sections separated by language.
+    Prints both the usable and unusable, relevant bears
+    in sections indexed by language.
 
     :param printer:
         A ``ConsolePrinter`` object used for console interactions.
     :param relevant_bears:
-        A dict with language name as key and bear classes as value.
+        An ``OrderedDict`` indexed by “usable” and “unusable” bears stored in
+        dictionaries that use language as key and bear classes as value.
     """
-    if label == 'relevant':
-        printer.print(BEAR_HELP)
-
-    printer.print('\nBased on the languages used in project the following '
-                  'bears have been identified to be %s:' % label)
-    for language in relevant_bears:
-        printer.print('    [' + language + ']', color='green')
-        for bear in relevant_bears[language]:
-            printer.print('    ' + bear.name, color='cyan')
-        printer.print('')
+    printer.print(BEAR_HELP)
+
+    # Don't print anything for empty bear sets
+    nonempty_label_bears = (
```

Comment:
Should we print something if there are no relevant bears?

Reply:
i am sure that would never be the case 😉

Reply:
yeah, imo a simple print statement saying no relevant bears found should suffice (just in case there's a change of design in future which actually allows no relevant bears, we don't want user to be confused when he sees an empy screen with no message printed)